Generator/Generator/MethodGen.swift

```diff
@@ -209,7 +341,42 @@ func methodGen (_ p: Printer, method: MethodDefinition, className: String, cdef:
             if args != "" { args += ", "}
             args += "_ arguments: Variant..."
         }
+        // REFACTOR STEP 1:  generate a helper function, a la withUnsafePointers() above, which
+        // combines extracting the parameters into pointers and packing them into the _args array.
+        // We can modularize this by creating functions that generate the return type and return
+        // statements.
+        #if true
```

Comment:
I am curious, what is the goal for step 2 in this refactoring, and do we need the old code that is on the `#else` part?

Reply:
Hopefully we can retire the old code. Just kept it for comparision's sake. Hence the draft PR.